blog/content/post/2021-02-02-fosdem2021.md

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
There are two the same files, but with different publication dates. Why?

Reply:
I tried to change the date in the file.

Reply:
I removed duplicated file